pkg/cdc/generate_config.go

```diff
@@ -91,9 +91,12 @@ func updateAdditionalJVMOpts(optsSlice []string, CDCConfig *cassdcapi.CDCConfigu
 		}
 	}
 	CDCOpt := fmt.Sprintf("-javaagent:%s=%s", "/opt/cdc_agent/cdc-agent.jar", strings.Join(optsSlice, ","))
+	// We want to disable MCAC when the server being deployed is DSE
```

Comment:
We also want to remove MCAC agent when MGMT_API_DISABLE_MCAC is set (like done in https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/master/scripts/docker-entrypoint.sh#L43)

Does this process mess up with that logic?

Reply:
Maybe, but that's a problem to solve under that PR, not this one IMO. That one is still under development anyway.

Reply:
Sorry, I meant [this](https://github.com/k8ssandra/k8ssandra-operator/pulls) PR.

Reply:
I created #481 to track this. We need it also.

Reply:
(separate PR)